docs/tutorials/visibility_simulator.ipynb

```diff
@@ -57,12 +57,22 @@
   },
```

Comment:
<p>In the "subband" entry of the <code>spectral_type</code> explanation, "... band centers give in ..." should be "... band centers given in ...". In the second-to-last paragraph, I think we want to replace "VisCPU and healvis wrappers" with "visibility simulator wrappers". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Done.</p>